wagtail/wagtailadmin/utils.py

```diff
@@ -19,6 +21,30 @@
 
 logger = logging.getLogger('wagtail.admin')
 
+# Wagtail languages with >=90% coverage
+# This list is manually maintained
+WAGTAILADMIN_PROVIDED_LANGUAGES = [
+    ('en', _('English')),
+    ('de', gettext_lazy('German')),
+    ('pt-br', gettext_lazy('Brazilian Portuguese')),
+    ('es', gettext_lazy('Spanish')),
+    ('ro', gettext_lazy('Romanian')),
+    ('fr', gettext_lazy('French')),
+    ('is-is', gettext_lazy('Icelandic')),
+    ('it', gettext_lazy('Italian')),
+    ('nb', gettext_lazy('Norwegian Bokmål')),
+    ('pl', gettext_lazy('Polish')),
+    ('pt-pt', gettext_lazy('Portuguese')),
+    ('ru', gettext_lazy('Russian')),
+    ('nl-nl', gettext_lazy('Netherlands Dutch')),
+    ('fi', gettext_lazy('Finish')),
+    ('ga', gettext_lazy('Galician'))
+]
+
+
+def get_available_admin_languages():
+    return getattr(settings, 'WAGTAILADMIN_PERMITTED_LANGUAGES', WAGTAILADMIN_PROVIDED_LANGUAGES)
```

Comment:
Shouldn’t we combine `WAGTAILADMIN_PROVIDED_LANGUAGES` with `LANGUAGES`?
It feels irrelevant to propose Russian and Icelandic in a site where editors are supposed to type (and therefore know) only French & English (in the cases I see daily).

Another good reason to combine with `LANGUAGES`: if a clumsy beginner user selects the wrong language, he might end up with the back office in a language and alphabet he doesn’t know, so he may not find again how to change it back.